src/main/java/com/cascadebot/cascadebot/commands/core/InviteCommand.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2018 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package com.cascadebot.cascadebot.commands.core;
+
+import com.cascadebot.cascadebot.CascadeBot;
+import com.cascadebot.cascadebot.commandmeta.CommandContext;
+import com.cascadebot.cascadebot.commandmeta.CommandType;
+import com.cascadebot.cascadebot.commandmeta.ICommand;
+import net.dv8tion.jda.core.Permission;
+import net.dv8tion.jda.core.entities.Member;
+
+public class InviteCommand implements ICommand {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        public static String getInvite() {
```

Comment:
this isn't how methods work.......

Reply:
^^^ This needs to be in the main file also